tests/helpers/start-app.js

```diff
@@ -1,6 +1,7 @@
 import Ember from 'ember';
 import Application from '../../app';
 import config from '../../config/environment';
+import './simple-auth';
```

Comment:
This shouldn't be necessary as everything under test-support should be merged into the app tree anyway.